README.md

```diff
@@ -1888,6 +1888,8 @@ NOTE: `callback` is an optional URL for pushover to [ack messages](https://pusho
 The pushover service will accept a payload with either a simple text message, or a json payload which contains
 a `message` and either an `imageurl` or `imagebase64` encoded image.
 
+Further, the imageurl payload, can have the addition parameters of an auth type (basic, digest) and a user and password.
+
```

Comment:
Please elaborate what this is actually used for. Why should I add this?

Reply:
I've added some context as to why.

Reply:
I've added some context to why this might be useful.

Sorry I can't figure out how to add the relevant code section that was updated in the last commit.